OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/preferencecenter/OptimovePreferenceCenter.java

```diff
@@ -0,0 +1,281 @@
+package com.optimove.android.preferencecenter;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.optimove.android.Optimove;
+import com.optimove.android.OptimoveConfig;
+import com.optimove.android.main.common.UserInfo;
+import com.optimove.android.main.tools.networking.HttpClient;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import okhttp3.Response;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class OptimovePreferenceCenter {
+    private static final String TAG = OptimovePreferenceCenter.class.getName();
+    private static OptimovePreferenceCenter shared;
+    private static Config config;
+
+    static ExecutorService executorService;
```

Comment:
last q here from code perspective is, like Chris was saying, creating too many executors is not great -- each executor manages own work queue, runs on own tread. This can lead to too much memory used / CPU spending time switching threads VS doing useful work. In android sdk we already have 3 executors, ideally we mb could reuse executors set in 'core'

but not in this PR anyway. also setting preferences is mb a rare niche task, mb should affect it all too badly